frontend/src/metabase/collections/containers/FormCollectionPicker/FormCollectionPicker.tsx

```diff
@@ -118,6 +102,14 @@ function FormCollectionPicker({
     [id, value, type, title, placeholder, error, touched, className, style],
   );
 
+  const formik = useFormikContext();
```

Comment:
You use only `values` from it, it would look a bit more concise if you destructured it here like `const { values } = useFormikContext();`

Reply:
90599e794d